crates/control/src/behavior/stand.rs

```diff
@@ -11,13 +11,16 @@ use types::{
 pub fn execute(
     world_state: &WorldState,
     field_dimensions: &FieldDimensions,
+    role: &Role,
 ) -> Option<MotionCommand> {
     match world_state.robot.primary_state {
         PrimaryState::Initial => Some(MotionCommand::Stand {
             head: HeadMotion::ZeroAngles,
         }),
         PrimaryState::Set => {
             let ground_to_field = world_state.robot.ground_to_field?;
+            let mut kicking_team_in_set = Team::Uncertain;
```

Comment:
I would prefer this to be a separate match on the filtered_game_controller_state only checking the required variant instead of having mutable variables.

Reply:
Done